index.ts

```diff
@@ -340,10 +347,70 @@ cron.schedule("0 * * * *", async () => {
     }
   } catch (err) {
     console.log("🚨🚨 Error in unsnooze cron job 🚨🚨");
-    console.log(err);
+    console.error(err);
   }
 });
 
+cron.schedule(
+  "0 9,16 * * *",
+  async () => {
+    console.log("⏳⏳ Running status report cron job ⏳⏳");
+    try {
+      for await (const maintainer of MAINTAINERS) {
+        const files = readdirSync("./config");
+        let text = `:wave: Hey ${maintainer.id}, here's your daily status report!`;
+        const user = await prisma.user.findFirst({
+          where: { OR: [{ slackId: maintainer.slack }, { githubUsername: maintainer.github }] },
+        });
+
+        if (!user || user.optOut) continue;
```

Comment:
@faisalsayed10 We can optimise this by bringing read operation and DB query outside of `for...of` loop:

```js
      const files = readdirSync("./config");
      const users = await prisma.user.find({
        where: { 
          OR: [
                    { slackId: { in: MAINTAINERS.map(maintainer => maintainer.slack) } },
                    { githubUsername: { in: MAINTAINERS.map(maintainer => maintainer.github) } }
          ],
          optOut: false
        },
      });

      for await (const user of users) {
        const maintainer = MAINTAINERS.find(maintainer => maintainer.slack === user.slackId)
        let text = `:wave: Hey ${maintainer.id}, here's your daily status report!`;

        for await (const file of files) {
        ...
```